src/server/dcp_server/test_integration.py

```diff
@@ -0,0 +1,100 @@
+import os
+import cv2 
+import sys
+import torch 
+
+import numpy as np
+from tqdm import tqdm
+from copy import deepcopy
+from skimage.color import label2rgb
+
+sys.path.append("../")
+from dcp_server.models import CellposePatchCNN
+from dcp_server.utils import read_config
+
+import pytest
+
+
+def get_dataset(dataset_path):
+
+    images_path = os.path.join(dataset_path, "images")
+    masks_path = os.path.join(dataset_path, "masks")
+
+
+    images_files = [img for img in os.listdir(images_path)]
+    masks_files = [mask for mask in os.listdir(masks_path)]
+
+    images, masks = [], []
+    for img_file, mask_file in zip(images_files, masks_files):
+
+        img_path = os.path.join(images_path, img_file)
+        mask_path = os.path.join(masks_path, mask_file)
+
+        img = cv2.imread(img_path, cv2.IMREAD_GRAYSCALE)
+        msk = np.load(mask_path)
+
+        images.append(img)
+        masks.append(msk)
+
+    return images, masks
+
+
+@pytest.fixture
+def patch_model():
+    classifier_model_config = {
+        "chpt_path":
+        "/home/ubuntu/data-centric-platform/src/server/dcp_server/data/classifier_checkpoint.pth"
```

Comment:
We cannot have hard coded paths in the tests (only the configs can have hard coded paths), this should work across platforms. Why do we need to load a pertained model? You are training the model anyway in test_train_run right?

Reply:
Yes, we train it from scratch, changed it.